apps/monitoring/kube-prometheus-stack-crds-upgrade/kustomize.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
```

Comment:
This has been changed to v1 upstream. See https://github.com/hmcts/cnp-flux-config/blob/8dc8c7f27dd1303e424a494a46934755cfdc79d1/apps/monitoring/base/kustomize.yaml#L16

Reply:
Ah I see thank you will fix :)